### PR TITLE
Migration to Python3

### PR DESCRIPTION
**Pull request by jova2**
Hi,

I tested this code on python3 and it seems stable to me.

Please merge this into master and release version 2.9.

I will become the speedometer maintainer in Debian and will add this code as a runtime patch. I will be happy to update the package with this update as it becomes available.

cheers

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,10 +1,19 @@
 Speedometer
 ===========
 
+Fork from <https://github.com/wardi/speedometer>
+
```
Comment (wardi) [inline README.rst]: no longer a fork if this gets merged :)

File: speedometer.py [hunk 2]
```diff
@@ -442,8 +449,8 @@ def local_maximums(self, pad, left):
         for i in range(left+max(0, ldist-pad),len(l)-rdist+1):
             li = l[i]
             if li == 0: continue
-            if i and l[i-1]>=li: continue
-            if l[i+1]>li: continue
+            if i and l[i-1] != None and l[i-1]>=li: continue
```
Comment (wardi) [inline speedometer.py]: `is not None` is the preferred test for None

File: speedometer.py [hunk 3]
```diff
@@ -495,7 +502,7 @@ def readable_speed(speed):
     if speed == None or speed < 0: speed = 0
 
     units = "B/s  ", "KiB/s", "MiB/s", "GiB/s", "TiB/s"
-    step = 1L
+    step = long(1)
```
Comment (wardi) [inline speedometer.py]: I don't think there was a reason to use 1L here (unless we're relying on the repr), how about using normal ints throughout and let python2 upgrade them as required

**Comment by wardi** [comment]
only a few small comments otherwise good to go

**Comment by jova2** [comment]
@kenduest could you also comment on this?

**Comment by wardi** [comment]
Also, wouldn't it be simpler to drop python2 support? This is a stand-alone application, no-one should care that they need python3 to run it, right?

**Comment by jova2** [comment]
Sorry, I can't opine on this, I tested the @kenduest code in a python3 environment and it's OK.
I believe these changes fix the bug https://bugs.debian.org/943229 so it is interesting a 2.9 release with these fixes.

**Pull request merged**